uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/bookingcheckout/activity/BookingCheckoutActivity.kt

```diff
@@ -0,0 +1,154 @@
+package com.karhoo.uisdk.screen.booking.booking.bookingcheckout.activity
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import com.karhoo.sdk.api.model.Quote
+import com.karhoo.sdk.api.network.request.PassengerDetails
+import com.karhoo.uisdk.KarhooUISDK
+import com.karhoo.uisdk.R
+import com.karhoo.uisdk.base.BaseActivity
+import com.karhoo.uisdk.screen.booking.booking.bookingcheckout.fragment.BookingCheckoutFragment
+import com.karhoo.uisdk.screen.booking.booking.bookingcheckout.views.BookingCheckoutViewContract
+import com.karhoo.uisdk.screen.booking.booking.payment.PaymentActions
+import com.karhoo.uisdk.screen.booking.domain.address.BookingStatus
+import kotlinx.android.synthetic.main.uisdk_activity_base.khWebView
+import java.util.*
+
+class BookingCheckoutActivity : BaseActivity(), PaymentActions {
```

Comment:
Renamed files to CheckoutActivity | CheckoutFragment | CheckoutView | CheckoutViewContract | CheckoutViewPresenter.
Regarding moving it into a separate package, not really sure because it's related to a/the booking flow